internal/mongoerrors/mongoerrors.go

```diff
@@ -35,6 +35,8 @@ import (
 	"github.com/FerretDB/FerretDB/v2/internal/util/must"
 )
 
+//nolint:godot // skip temporary
+// TODO https://github.com/FerretDB/FerretDB/issues/4696
```

Comment:
The linter fails like below
https://github.com/FerretDB/FerretDB/actions/runs/12981056822/job/36198940808
```
Error: internal/mongoerrors/mongoerrors.go:38:1: Sentence should start with a capital letter (godot)
// skip for now
^
task: Failed to run task "lint": exit status 1
```